janusgraph-test/src/test/java/org/janusgraph/graphdb/database/IndexSerializerTest.java

```diff
@@ -125,6 +126,7 @@ private JanusGraphElement mockIndexAppliesTo(MixedIndexType mit, boolean indexab
 
     private JanusGraphElement mockIndexableElement(String key, String value, boolean indexable) {
         StandardJanusGraphTx tx = mock(StandardJanusGraphTx.class);
+        when(tx.getNextTx()).thenReturn(tx);
```

Comment:
Nit: We are seemingly using `doReturn(...).when(...)` everywhere else.
```suggestion
        doReturn(tx).when(tx).getNextTx();
```

Reply:
This would also avoid the extra import